scripts/byobu_startup.py

```diff
@@ -172,37 +209,37 @@ def setup_main_control_window(cmd_prefix='#'):
     # setup the main-control window
     select_window(WINDOWS[0])
     # Select default pane. Probably an unnecessary line of code
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split window hoirzontaly
-    subprocess.call(f"byobu split-window -h", shell=True)
+    call_byobu(f"split-window -h")
     # select pane 0
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split selected pane vertically
-    subprocess.call(f"byobu split-window -v", shell=True)
+    call_byobu(f"split-window -v")
     # select the top pane
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split top pane vertically again
-    subprocess.call(f"byobu split-window -v", shell=True)
+    call_byobu(f"split-window -v")
     # select the top pane
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
 
     clear_current_pane()
     # Now run the necessary commands in each pane
     send_command_to_pane(
-        cmd_prefix + '$HUNTSMAN_POCS/scripts/pyro_name_server.py', 0)
+        cmd_prefix + '${HUNTSMAN_POCS}/scripts/pyro_name_server.py', 0)
 
     send_command_to_pane(
-        cmd_prefix + 'python $HUNTSMAN_POCS/scripts/start_config_server.py', 1)
+        cmd_prefix + 'python ${HUNTSMAN_POCS}/scripts/start_config_server.py', 1)
 
-    send_command_to_pane(f'ipython', 2)
+    send_command_to_pane(f'jupyter-lab', 2)
```

Comment:
I'm not sure if we've decided this is what we want actually. Might be better to stick with `ipython`.